docs/modules/ROOT/pages/index.adoc

```diff
@@ -287,6 +287,37 @@ Edit the karma.conf.js:
 },
 ----
 
+[#vite]
+=== Vite
+Any app created with Vite (https://vitejs.dev/guide/) should work with Quinoa after the following changes (assuming the
+generated files are in folder `src/main/webui`):
+
+In application.properties add:
+[source,properties]
+----
+quarkus.quinoa.dev-server.port=5173
+quarkus.quinoa.build-dir=../webui/dist
```

Comment:
maybe i should drop the (assuming the generated files are in folder `src/main/webui`) as well then?

Reply:
I start the vite-section with 
"Any app created with Vite (https://vitejs.dev/guide/) should work with Quinoa after the following changes (assuming the generated files are in folder `src/main/webui`):"

The only reason I added the assuming-part was because I used "quarkus.quinoa.build-dir=../webui/dist", if we  use "quarkus.quinoa.build-dir=dist" instead, maybe we should shorten it down to: 

"Any app created with Vite (https://vitejs.dev/guide/) should work with Quinoa after the following changes"